### DISCUSSION TITLE
Ideas para UbiCate: Propuesta, Desarrollo y Decisión

### DISCUSSION THREAD

**Discussion by vlermandac** [open-source-uc/UbiCate-v2#112] [Ideas]
# Foro de Ideas para UbiCate 📍
Bienvenidos al espacio dedicado a la propuesta, discusión y desarrollo de ideas para nuestro proyecto. El objetivo de este foro es facilitar un entorno colaborativo donde podamos compartir
  nuevas ideas, comentar sobre las ya existentes, y trabajar juntos para convertirlas en propuestas concretas y viables.

## Propósito
- **Proponer Ideas:** Comparte tus ideas innovadoras para mejorar o expandir el proyecto.
- **Discutir y Desarrollar:** Participa en la discusión de ideas propuestas por otros. Aporta para desarrollar una planificación para la potencial implementación de las propuestas.
- **Evaluar Factibilidad:** Finalmente, entre todos, evaluaremos la viabilidad y popularidad de cada idea para decidir si debe implementarse o no.

## Considerar
- Este foro nos permite explorar y refinar estas propuestas, que pueden ser muy amplias y ambiguas, antes de darles un rumbo concreto. Si tu idea está suficientemente desarrollada, tienes la libertad de plantearlo directamente como un *issue*.
- También es un espacio para replantear ideas que necesitan un enfoque diferente o más detallado antes de ser consideradas para implementación.

¡Gracias por tu participación!

**Comment by vlermandac** [comment]
## Ideas que quedaron del Trello (ahora en desuso)
- Agregar la posibilidad de tomar una foto de la sala en el formulario.
- Guardar lugares favoritos.
- Rellenar la info de las ubicaciones (a mano y/o scrapear lugares conocidos).
- Validar input formulario (imagino que cosas sin sentido como nombres de salas con emojis o cosas raras).
- Usar Protomaps para usar un  open source hosted map en caso de no usar mapbox https://docs.protomaps.com/pmtiles/maplibre
- Eventos temporales, ejemplo: ferias laborales, charlas, etc. Lo malo de esto es que puede ser difícil de mantener actualizado.
- Hacer que otras páginas de la uc puedan hacer embed de nuestro mapa (investigar como es posible).
- Mostrar los polígonos de los edificios en el mapa, ver si hay que añadir un botón para mostrar/ocultar y ver cómo agregarlos al buscador.

**Comment by pipeaw** [comment]
Sería útil fijar en el mapa íconos en puntos de referencia. e.g.  Baticristo, templo, etc.
